cmake/GinkgoConfig.cmake.in

```diff
@@ -81,6 +82,10 @@ set(GINKGO_DPCPP_FLAGS @GINKGO_DPCPP_FLAGS@)
 set(GINKGO_MKL_ROOT @GINKGO_MKL_ROOT@)
 set(GINKGO_DPL_ROOT @GINKGO_DPL_ROOT@)
 
+set(GINKGO_HAVE_MPI @GINKGO_HAVE_MPI@)
+set(GINKGO_HAVE_CUDA_AWARE_MPI @GINKGO_HAVE_CUDA_AWARE_MPI@)
+set(GINKGO_FORCE_CUDA_AWARE_MPI @GINKGO_FORCE_CUDA_AWARE_MPI@)
```

Comment:
```suggestion
set(GINKGO_HAVE_CUDA_AWARE_MPI @GINKGO_HAVE_CUDA_AWARE_MPI@)
```
What is the difference between BUILD_MPI and HAVE_MPI? I think they should be equivalent from the user side. Also whether we have autodetected or forced CUDA-aware MPI should be of no consequence for users as well.